backend/oasst_backend/api/v1/hugging_face.py

```diff
@@ -0,0 +1,30 @@
+from typing import List
+
+from fastapi import APIRouter, Depends
+from oasst_backend.api import deps
+from oasst_backend.models import ApiClient
+from oasst_backend.schemas.hugging_face import ToxicityClassification
+from oasst_backend.utils import get_detoxify_classification
+
+router = APIRouter()
+
+
+@router.get("/hf/text_toxicity")
+def get_text_toxicity(
+    msg: str,
+    hf_token: str,
+    api_client: ApiClient = Depends(deps.get_trusted_api_client),
+) -> List[List[ToxicityClassification]]:
+    """Get the Message Toxicity from HuggingFace Roberta model.
+
+    Args:
+        msg (str): the message that we want to analyze.
+        hf_token (str): the token for the huggingface access.
```

Comment:
The HF token could better be stored in the backend  configuration, e.g. as a member of our pydantic [Settings classs](https://github.com/LAION-AI/Open-Assistant/blob/main/backend/oasst_backend/config.py).